package-lock.json

```diff
@@ -32,6 +32,14 @@
         "gh-pages": "^5.0.0"
       }
     },
+    "node_modules/@aashutoshrathi/word-wrap": {
```

Comment:
Did you mean to add this module? If so, please add this to the [Knowledge Board](https://docs.google.com/document/d/1BXvILLekHlO0pEJhnBhob7JFHt3supxyvAGyhev482M/edit?usp=sharing) explaining what it does and why it's been added. Also, you downgraded a lot of the packages (probably from running **_npm install_**. There may be something that the backend will have to work on to fix.) You may have been getting that error before because of a dependecy with the module.